vent/coordinator/rpc.py

```diff
@@ -39,7 +42,25 @@ def get_control(control_setting_name):
     return pickle.dumps(res)
 
 
+def init_logger():
```

Comment:
exhausted right now, but imo this would be really easy to give this function an argument and use that to control the log filename rather than hardcoding it, then rather than needing to define it twice with minor alterations (and not use one like this in the gui at all) we define it once and use it everywhere?

Reply:
i'll do this tmrw

Reply:
I agree with your suggestions. This would make it cleaner.

Reply:
that sounds good! Upon startup, filenames could be generated in the `coordinator`, and handed to the python-logger, and the tables/hdf5-logger in the controller?

Reply:
no need to hand filenames around, can init logger with `__name__` as an argument and generate filename from that. i'll show u in a minute